rosetta-lang/src/main/java/com/regnosys/rosetta/validation/RosettaSimpleValidator.xtend

```diff
@@ -1051,11 +1054,11 @@ class RosettaSimpleValidator extends AbstractDeclarativeRosettaValidator {
 	
 	@Check
 	def checkConstructorExpression(RosettaConstructorExpression ele) {
-		val rType = ele.RType
-		if (rType !== null) {
-			val baseRType = rType.stripFromTypeAliases
+		val rMetaAnnotatedType = ele.RMetaAnnotatedType
```

Comment:
`val rType = ele.RMetaAnnotatedType.RType` reads a lot easier than doing all of that extraction below